wandb/sdk_py27/internal/sender.py

```diff
@@ -747,18 +771,22 @@ def send_output(self, data):
             self._fs.push(filenames.OUTPUT_FNAME, line)
             self._partial_output[stream] = ""
 
-    def _update_config(self):
+    def _update_config(self, data, debounce=False):
         config_value_dict = self._config_format(self._consolidated_config)
-        self._api.upsert_run(
-            name=self._run.run_id, config=config_value_dict, **self._api_settings
-        )
+        if debounce:
+            self._debounced_records.append(data)
+        else:
+            self._api.upsert_run(
+                name=self._run.run_id, config=config_value_dict, **self._api_settings
+            )
+            self._config_at_last_upsert = config_value_dict
+
         self._config_save(config_value_dict)
```

Comment:
Lets put config_save in the debouncer also.

Reply:
done